src/Blocks/BlockOre.h

```diff
@@ -50,6 +50,11 @@ class cBlockOreHandler :
 				a_Pickups.push_back(cItem(E_ITEM_COAL));
 				break;
 			}
+			case E_BLOCK_QUARTZ_ORE:
```

Comment:
Clearly. I'm the only man who could've done it. -cough-

Done that.

On Wed, Apr 22, 2015 at 2:48 PM, NiLSPACE <notifications@github.com> wrote:

> In src/Blocks/BlockOre.h
> <https://github.com/mc-server/MCServer/pull/1888#discussion_r28868257>:
>
> > @@ -50,6 +50,11 @@ class cBlockOreHandler :
> >  				a_Pickups.push_back(cItem(E_ITEM_COAL));
> >  				break;
> >  			}
> > +			case E_BLOCK_QUARTZ_ORE:
>
> Rename this to E_BLOCK_NETHER_QUARTZ_ORE
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mc-server/MCServer/pull/1888/files#r28868257>.
>